PyRDF/Node.py

```diff
@@ -187,13 +196,16 @@ def graph_prune(self):
         if not self.children:
             # Every pruning condition is written on
             # a separate line
-            if len(gc.get_referrers(self)) <= 3 or \
+            if len(gc.get_referrers(self)) <= self.REFERRERS_THRESHOLD or \
                (self.operation and self.operation.is_action() and self.value):
 
                 # ***** Condition 1 *****
+                #
+                # Python 2:
```

Comment:
As @etejedor suggested offline, I would put this explanation about why `REFERRERS_THRESHOLD` is different depending on the Python version on the docstrings of the class attribute. Here you can just leave a general explanation about the Condition 1.